import-export-cli/cmd/apis.go

```diff
@@ -80,7 +80,7 @@ var apisCmd = &cobra.Command{
 		//Since other flags does not use args[], query flag will own this
 		if len(args) != 0 && listApisCmdQuery != "" {
 			for _, argument := range args {
-				listApisCmdQuery +=  " " + argument
+				listApisCmdQuery +=  "&" + argument
```

Comment:
Shall we use the constant [1] here?

[1] https://github.com/wso2/product-apim-tooling/blob/3.2.x/import-export-cli/utils/constants.go#L124

Reply:
fixed with b736f4e7253bd8f35a7841654afe80b3a4acba3c